### PR TITLE
Feature setup partials

### PR DESCRIPTION
**Pull request by EstefaniaCK**
Hi Harsh, 

Please review and merge the pull request. 

Thanks,

### CODE REVIEW

**Comment by mandeepharsh** [comment]
Good work estefania

**Review by mandeepharsh — CHANGES_REQUESTED** [review]
We should download the fonts instead of importing them from google

File: src/styles/partials/color.scss
Comment (mandeepharsh) [inline src/styles/partials/color.scss:1]: We should not reference the color itself while naming it, instead we should refrence what that color will be used for, so if we need to change the color in the future we don't have to change the name of the variable everywhere

**Review by mandeepharsh — CHANGES_REQUESTED** [review]
@EstefaniaCK We should not reference the color itself while naming it, instead we should refrence what that color will be used for, so if we need to change the color in the future we don't have to change the name of the variable everywhere

**Comment by EstefaniaCK** [comment]
Hi Harsh, 

as discussed fonts have been downloaded and color names have been updated. 

If there is anything else please let me know

**Review by mandeepharsh — APPROVED** [review]
Great work!!

**Pull request merged**